### PR TITLE
Chapter1/exterminator

### PR DESCRIPTION
**Pull request by mihailfox**

### CODE REVIEW

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Check.cs  1
- Src/BootCamp.Chapter/lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886034&bid=16185853)

File: Src/BootCamp.Chapter/Check.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        {
+            return Lesson3.ReadMyInput(message);
+        }
+
+        public float PromptFloat(string message)
+        {
+            return Lesson3.AsFloat(message);
+        }
+
+        public float WhatIsTheBMI(float weight, float heigth)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Check.cs]: Codacy Issue found: [Rename method 'WhatIsTheBMI' to match pascal case naming rules, consider using 'WhatIsTheBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886034)

File: Src/BootCamp.Chapter/Check.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+namespace BootCamp.Chapter
+{
+    public class Checks
+    {
+        public static string PromptString(string message)
+        {
+            return Lesson3.PromptString(message);
+        }
+
+        public int PromptInt(string message)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Check.cs]: Codacy Issue found: [Make 'PromptInt' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886034)

File: Src/BootCamp.Chapter/Check.cs (hunk 1 above)
Comment (Almantask) [inline Src/BootCamp.Chapter/Check.cs]: Codacy Issue found: [Make 'WhatIsTheBMI' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886034)

File: Src/BootCamp.Chapter/Check.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        {
+            return Lesson3.ReadMyInput(message);
+        }
+
+        public float PromptFloat(string message)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Check.cs]: Codacy Issue found: [Make 'PromptFloat' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886034)

File: Src/BootCamp.Chapter/lesson3.cs [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                WriteLine("Invalid!");
+                return -1;
+            }
+            else
+            {
+                return float.Parse(input, CultureInfo.InvariantCulture);
+            }
+        }
+
+        public static float CalcBMI(float weight, float heigth)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/lesson3.cs]: Codacy Issue found: [Rename method 'CalcBMI' to match pascal case naming rules, consider using 'CalcBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886034)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Check.cs  1
- Src/BootCamp.Chapter/lesson3.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4886034&bid=16185853)

**Comment by ellmaster** [comment]
I think you did pretty well!
Though, there are a bit more points to "fix"/point out (running Code / no Codacy errors do not mean that this piece of code is good! :P)

Maybe you can also add some comments to the file what you did?
After all this challange is also for you to learn and remember things, if you justify your edits, you will probably keep it up in your head even more! :)

**Pull request closed without merging**